src/mocks/data/draft_reports_object.json

```diff
@@ -0,0 +1,50 @@
+{
+  "status": "processing", // enum: one of ["processing", "failed", "delivered"]
+  "current_stage": "dex-upload",
+  "current_action": "metadata-validation",
+  "file_name": "test.txt",
+  "transport_id": "unique_guid", //represents the id for the entire transaction i.e. upload_id
+  "timestamp": "12-12-2023", -- first initial timestamp
+  "data_stream_id": "aims-celr",
+  "data_stream_route": "csv",
+  "reports": [
+    {
+    "stage": "DEX HL7 v2", //describes a chunk of the pipeline, usually associated with a specific product; string could be human readable, or transformed from id to human readable
```

Comment:
why do we have a current_stage on top and a stage here?
same for action

Reply:
See above.